docs/arcadia-users-group/20220919-aws-s3-cli/lesson.md

```diff
@@ -0,0 +1,301 @@
+# How to use the AWS S3 command line tool
+
+## Accessing the shell for this lesson
+
+For this lesson, we need to have access to a Unix shell.
+If you're not sure how to open a terminal on your computer, see [these instructions](https://swcarpentry.github.io/shell-novice/setup.html).
+
+## What is the AWS S3 CLI and why do we care about it?
+
+Arcadia uses Amazon Web Services (AWS) S3 to store our files remotely, sometimes as backup, sometimes to host files temporarily before we deposit them to a FAIR data repository.
+
+Some of us may already know how to interact with AWS S3 through its own user interface or using a tool like [Cyberduck](https://www.notion.so/arcadiascience/Syncing-data-to-S3-via-a-GUI-for-the-microscopy-team-bd7c106e648343d4ac01c2d5198b633c). While these options work fine, they limit the speed at which you can interact with S3 (they put explicit caps on upload/download speeds) and may be burdensome for large scale changes (think: updating 100s of files).
+
+This is where the AWS S3 command line tool comes into play. During this workshop, we'll learn about two tools: the official [AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/s3/) and [s5cmd](https://github.com/peak/s5cmd).
+
+Command line tools may be intimidating but great news: if you attended the AUG workshops in the last 2 weeks focusing on shell commands, you already know the shell versions of all the commands (`ls`, `cp`, `mv`, `rm` etc.) that we'll go through in this workshop. So let's get started!
+
+## Downloading and installing the AWS CLI
+
+The download and installation instructions for the AWS CLI can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). If you're on a Unix machine, you can use these two commands to start the installation process:
```

Comment:
I think these are only the instructions for Linux/WSL? Would these work for mac too? The linked URL looks like the macOS instructions are different